src/desktop/apps/artist/routes.coffee

```diff
@@ -36,6 +38,9 @@ sd = require('sharify').data
 
       return res.redirect(artist.href) unless(_.find nav.sections(), slug: tab) or artist.counts.artworks is 0
 
+      artistTest = new SplitTest req, res, runningTests.artist_page
```

Comment:
I don't think you need to do this. By setting up the split test, you automatically create a sharify var that will contain the value.

so in this case, `sd.ARTIST_PAGE` can be checked against 'control', etc.

So you don't have to finagle it here. I might recommend changing the name, `sd.ARTIST_PAGE` is not super descriptive.

Reply:
Got it, thanks for the heads up. I'll update the name as well